.github/workflows/db_test.yaml

```diff
@@ -0,0 +1,32 @@
+name: run-db-test
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  db_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+
+      - name: Install Python Dependencies
+        run: pip install -r requirements.txt
+        
+      - name: Set PYTHONPATH
+        run: echo "PYTHONPATH=$(pwd):$(pwd)/amos2023ss03-qachat" >> $GITHUB_ENV
+
+      - name: Run DB Test
+        env:
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL}}
+          SUPABASE_SERVICE_KEY: ${{ secrets.SUPABASE_SERVICE_KEY}}
+
+        working-directory: ./Testing
+        run: python db_intergration_test.py
```

Comment:
As we already have a workflow file, why don't you combine it? As we there already set up Python and installed the dependencies.  So no need to do this twice and increase the runtime of the CI.

Reply:
I thought about separation of concerns. but actually you are right. I'm going to join them together